internal/internal_task_handlers.go

```diff
@@ -664,6 +664,10 @@ func (wth *workflowTaskHandlerImpl) ProcessWorkflowTask(
 		return nil, errors.New("nil or empty history")
 	}
 
+	if task.Query != nil && len(task.Queries) != 0 {
```

Comment:
Will we be able to clean this up (removing Query) when we have the new one implemented?